src/Enzo/hydro-mhd/EnzoMethodPpm.cpp

```diff
@@ -265,7 +288,7 @@ double EnzoMethodPpm::timestep ( Block * block ) throw()
 			&enzo_block->CellWidth[0],
 			&enzo_block->CellWidth[1],
 			&enzo_block->CellWidth[2],
-			&gamma, &EnzoBlock::PressureFree[in], &cosmo_a,
+			&gamma, &pressure_free_int, &cosmo_a,
```

Comment:
I think this change means you can remove the line setting `in`

Reply:
good catch! I removed it!